Integrations/integration-cisco-meraki.yml

```diff
@@ -4,7 +4,7 @@ commonfields:
AAAAAAAAAAAADbhjALAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAbBvCLAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAALBtCLMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADAtiHMAgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAANuGMAsAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABsG8IsAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAsG0IswAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAMC2IcwCAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA28ZzCgAAAAAAAHBry7Nnqfw1/ePqds1kte369S/Ipt8vZzFGX2X61xtdpX/L+uG4WeJs+/C6boXpt7f7S1X2zqm8ZXmFPp0Tce1DJfr4cTndf+R9dxsd/Mod3GRpp4vjXbIS5m0z2WXidM5I3C1xOm9M8u0RQzbVddnXh9rnS1KNlmI9dyhXc8txz8J1Rw7c9+vysvt/oT3uoeHbLYtMum/c/4C3eoIkxSjGu/I8tOut0f2SpPYDO2Pbr6Tf2R19jsv6vHaFdF9Jz+P6fdOG19pjXJfVrFwycwEAAAAAAAAAAG5Tcuge+zpSV2Xzs6rQrOw0lN36xdTXk1zu9zSb1f2G53lWAcx5qOttKOFlWVcrK891fV/H21DLG3rjvVwhB+Q9f33n+cs/eZY7csOCiXnOhcm8xLDLpmaHhGaHjdNdkuJYjJskVx0SXy0lXx8WP16KfnooZ384jkeHjhw456vy4h/6suyXG/u307cNy+1z07+pLV9Vi3dDIXP1+6+t03X1yLXFua6+V2qh+oXzhusw2zVtfpnWrfdiuHMBAAAAAABwEjE5Z84CAAAAAAAAblXbFWaRN4QqlBiLlMpxrbVyrCYvFWIQ73x7nMqGmJP2ijl7v7P8nfacs2Ouubuv3F19ne5jrD/L1n6fODNudzClA82OXE6uvLlmauT2zdpHm6UcJ0nU9233S6H/CvrBs8sSrYS8bMKRFFdW/iVX/tKQ3JemK80/HVk6/CX56me+lHO+dmPT2hA+MXybIWwip+F7dudBv7uel3Ju9BzpfqUJLvePtpzLvCGLgjALAAAAAAAAAACA27kTHGYxhDqsDV/oalt2yKHvw+tl/Rt0byLZdUkS/qzzvs3vqr5zXIezRj7fzTXTexrr7mLGe87Scl9Jg9cPYUoRzYh1Q0SGHn9W3+vrd3b27dKKlZSXco7WhOgkp2mOcnX7Ef85SP7CtJEvH15e/qLceJd/ml7xe1d605+TWf3SltqdZtBvDJ3P/ZcxQypHYbu6Zho+4vrzS5gFAAAAAAAATkWEWQAAAAAAAODWl1djGTSooZktdVMozXrcTBfkEPtFv6FJbqB9Xtau9rRtbHIr8Rb9ccYiVftQlw2v+Oh3LHz+w+fVk2890jfL5zYmLWRXdxM8lUAII67kQVhxujKksilW6z/A8JtbMt1zDYMYAiXU0H9m2uWy3Yzah2yikRJGEdrz0TSNzI/HOVu5POxc/MuVxTP/4vAFL/qYXDC6rH1pM22Hly5gIqfQ9995Sf3JsXbNd+6fxw2nvXyuIdii7HucoRQbmhJpigMAAAAAAAAAALiNOVYh6Fit50eFVnTShlB0N2zMsa9RbaxPhdkRcrs9itdiXt1/kpG89yP3XPzUXz6s+tbVj3RLh/99mE4PVNWcNCGWMIjaVxJCEOdTqcPlvl6XretD4bvQDK3grQ2FzzGt+x4++bJe99R9vHfdcuqO62wS077GpJByMt+M411/u7zv7D87/Ohn/b1ctP9f2kMsL4tMSgx9PrquGdacVy3X+f599Ix14RdDKEa1GvRv19b7UldH3XD+8iyknjB6AAAAAAAA3PoIswAAAAAAAMCtbxvCLLRxzNr1TVkhZnHOlMOXTbl/K9cdo7r/o+4+H/NDdvidj3YSHiEu7hc3Tmbs58TXLnjjNMwizo5txYRUGsoqnRWqn1qqscnOGt3639t0/2zNujCL8lVLM1zomt76zxuD7UIs+iY1fV23IbYfNYjVZr7pyhFZOTKNIfxjWDn8sWu/dc1H5OBX/yqGabM6O5Mv31HDPPRjONef3v64G8MsdI1d24FImAUAAAAAAAAAAADUiQ6zkK6gpWHyGtoemkZ8pRn0Yfa+JbN9V3Vg7l4PfPgOsY/2vn648aOzzHg8FjfnxHobku7pxTo/C6VwzklMK91x+npd1HiHvjZX9lvzubp6Xr/ddGESGmZRtvVhGCnFWfhF+R59mIXEJpU3i9FIE9rPJEupka+HdMX/OnQk/M/p5LS/CV/+0FUudt9Jo/L1beKG8+pkCMwf6op9ZY8wCwAAAAAAAJzCCLMAAAAAAADASeNYPXFDU5bL/U79jqEPa/D9uo1ZDEa7wfqGMn3qrFh5w4133fOJ1/3waPngRamZnFOa0VzsZ2bqGtBC6JrEfDWS6XRaGt5Kk9oQ4jA8DsEV/cxNG6X+g5q8GrZRmtxSnn3j0hzXr9d9ymxO7bK+p76+e50mfEQxNpcwDV+bGENwrv3gYX7vh5bOvN/7j7zhWX8iGnkhcqSW1J3D8vlcmdnJ9J+jfN6+mbCb1ekmwixuJq1iOA+EWQAAAAAAAAAAANy2zMpER7Wa96EVRxWINg9V2Gxbd8yw7g2mUsmoy3TYJcuya/clb/mBucs++2N26boHiR+ZbI1dbqJY72afr9S6rNYQTXuYKF6LV30YhWQ/C5EvdbjYvc+svrUmiH5dvSx19btoVwPsS2h8XA2zKPW8dp+qqiSmpmyrrOv362L7TTVX9nG5ETPa8YkjC/vfffjZr/mAPFQOhiTJ9Wn8OYeuXuiq7u2DPjcy6T+S70Mrui9sjzqlsT/PtpzKNWkXnnsYAAAAAAAAtz7CLAAAAAAAAHDSuKVhFrZv6iozI7X/TH8E03WxiXHWeH/O+XvOXvhJN7fzYWZuxz7ZuejF124ymUjlu6CJJnbv5F
```

Comment:
This image is much bigger that the previous one, are you sure this is the correct one ?